benchmarks/core/arithmetic-series.bril

```diff
@@ -0,0 +1,9 @@
+@main(n: int) {
```

Comment:
You'll need an `# ARGS: ...` comment here so that Turnt provides the argument to your `@main` function via the command line.